qa/rpc-tests/test_framework/util.py

```diff
@@ -227,6 +228,25 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     proxy.url = url # store URL on proxy for info
     return proxy
 
+def assert_start_raises_init_error(i, dirname, extra_args=None, expected_msg=None):
+    with tempfile.SpooledTemporaryFile(max_size=2**16) as log_stderr:
+        try:
+            node = start_node(i, dirname, extra_args, stderr=log_stderr)
+            stop_node(node, i)
+        except Exception as e:
+            #assert 'bitcoind exited' in str(e) #node must have shutdown
```

Comment:
This will fail in zcash so i have it commented by now.

Reply:
This failure is caused by https://github.com/bitcoin/bitcoin/pull/7744 not having been backported. I've opened #4411 which backports that PR, along with https://github.com/bitcoin/bitcoin/pull/9832 and two other PRs that removed merge conflicts.

Reply:
Yea, this problem is gone now. Thanks.